File: replace.py
import os
import glob

files = glob.glob('src/components/**/*.tsx', recursive=True) + glob.glob('src/app/**/*.tsx', recursive=True)

for filepath in files:
    with open(filepath, 'r') as f:
        content = f.read()
    
    original = content
    content = content.replace('blue-', 'pink-')
    content = content.replace('#2563eb', '#ef8b92')
    
    if content != original:
        with open(filepath, 'w') as f:
            f.write(content)
        print(f"Updated {filepath}")

File: replace.js
const fs = require('fs');
const path = require('path');

function walk(dir, callback) {
    fs.readdir(dir, function(err, files) {
        if (err) throw err;
        files.forEach(function(file) {
            var filepath = path.join(dir, file);
            fs.stat(filepath, function(err, stats) {
                if (stats.isDirectory()) {
                    walk(filepath, callback);
                } else if (stats.isFile() && (filepath.endsWith('.tsx') || filepath.endsWith('.ts'))) {
                    callback(filepath);
                }
            });
        });
    });
}

walk('./src', function(filepath) {
    let content = fs.readFileSync(filepath, 'utf8');
    let original = content;
    
    // Replace standard Tailwind classes
    content = content.replace(/blue-/g, 'pink-');
    // Replace hex codes for the specific blue
    content = content.replace(/#2563eb/g, '#ef8b92');
    
    if (content !== original) {
        fs.writeFileSync(filepath, content);
        console.log(`Updated ${filepath}`);
    }
});
